src/main/java/org/folio/sample/controller/BookController.java

```diff
@@ -36,18 +38,16 @@ public ResponseEntity<List<BookDTO>> getAllBooks(Boolean onlyLeapYears) {
 
     if (Boolean.TRUE.equals(onlyLeapYears)) {
       return new ResponseEntity<>(
-        bookService
-          .getAllBooksPublishedInLeapYears()
-          .stream()
-          .map(bookMapper::toDto)
-          .toList(),
-        HttpStatus.OK
-      );
+          bookService
```

Comment:
I hate to nitpick, but the FOLIO standard is two spaces. You should be able to setup vscode/prettier to handle this for you, see https://github.com/ualibweb/ua-folio-docs/tree/main/docs/style and https://github.com/ualibweb/ua-folio-docs/tree/main/config/VSCode